frontend/src/app/testResults/uploads/UploadForm.tsx

```diff
@@ -163,7 +162,21 @@ export function getGuidance(error: EnhancedFeedbackMessage) {
   }
 }
 
-const Uploads = () => {
+interface UploadFormProps {
+  uploadResults: (file: File) => Promise<Response>;
+  alert?: JSX.Element;
+  uploadType: "Agnostic" | "Disease Specific";
+  spreadsheetTemplateLocation: string;
+  uploadGuideLocation?: string;
+}
+
+const UploadForm: React.FC<UploadFormProps> = ({
+  alert,
+  uploadResults,
+  uploadType,
+  spreadsheetTemplateLocation,
+  uploadGuideLocation,
+}) => {
   useDocumentTitle("Upload spreadsheet");
```

Comment:
maybe we add the `uploadType` or something like that to the for the document title so it will be unique